recaf-ui/src/main/java/software/coley/recaf/services/info/summary/builtin/MinecraftModSummarizer.java

```diff
@@ -0,0 +1,225 @@
+                        }
+                    }
+                }
+
+                if (json.has("depends")) {
+                    JsonObject depends = json.getAsJsonObject("depends");
+                    if (depends.has("minecraft")) {
+                        mcVersion = depends.get("minecraft").getAsString();
+                    }
+                }
+            } catch (Exception e) {
+                // Ignore JSON parsing errors
+            }
+
+            String finalMcVersion = mcVersion;
+            batch.add(() -> {
+                Label title = new BoundLabel(Lang.getBinding("service.analysis.is-fabric-mod"));
+                consumer.appendSummary(title);
+
+                if (!finalMcVersion.isEmpty()) {
+                    consumer.appendSummary(new Label(Lang.getBinding("service.analysis.minecraft-version").get() + " " + finalMcVersion));
+                } else {
+                    consumer.appendSummary(new BoundLabel(Lang.getBinding("service.analysis.minecraft-version-unknown")));
+                }
+
+                if (!mainClasses.isEmpty()) {
+                    consumer.appendSummary(new BoundLabel(Lang.getBinding("service.analysis.entry-points")));
+                    for (String mainClass : mainClasses) {
+                        // Try to find the main class in JVM class bundle
+                        JvmClassInfo classInfo = findClassInResource(resource, mainClass);
+                        if (classInfo != null) {
+                            // Found class, create label with icon
+                            String classDisplay = textService.getJvmClassInfoTextProvider(workspace, resource,
+                                    resource.getJvmClassBundle(), classInfo).makeText();
+                            Node classIcon = iconService.getJvmClassInfoIconProvider(workspace, resource,
+                                    resource.getJvmClassBundle(), classInfo).makeIcon();
+                            Label classLabel = new Label(classDisplay, classIcon);
+                            classLabel.setCursor(Cursor.HAND);
+                            classLabel.setOnMouseEntered(e -> classLabel.getStyleClass().add(Styles.TEXT_UNDERLINED));
+                            classLabel.setOnMouseExited(e -> classLabel.getStyleClass().remove(Styles.TEXT_UNDERLINED));
+                            classLabel.setOnMouseClicked(e -> actions.gotoDeclaration(workspace, resource,
+                                    resource.getJvmClassBundle(), classInfo));
+                            consumer.appendSummary(classLabel);
+                        }
+                    }
+                } else {
+                    consumer.appendSummary(new BoundLabel(Lang.getBinding("service.analysis.entry-points.none")));
+                }
+            });
+
+        }
+
+        // 2. Try to find Forge mod information
+        FileInfo forgeFileInfo = resource.getFileBundle().get("mcmod.info");
```

Comment:
I think `mcmod.info` is used mostly in older versions of forge _(Like `1.12`)_, and newer versions have this info at `META-INF/mods.toml` - Would be nice to support both